internal/tfengine/tfengine_test.go

```diff
@@ -33,7 +37,9 @@ func TestExamples(t *testing.T) {
 	}
 
 	for _, ex := range exs {
+		ex := ex // capture range variable
```

Comment:
I don't understand what this does, could you elaborate?

Reply:
Added a link to doc